common/src/main/java/com/github/khanshoaib3/minecraft_access/utils/system/KeyUtils.java

```diff
@@ -93,6 +93,10 @@ public static boolean isRightAltPressed() {
         return isAnyPressed(GLFW.GLFW_KEY_RIGHT_ALT);
     }
 
+    public static boolean isLeftOrRightAltPressed() {
```

Comment:
This method name is a bit verbose, `isAltPressed` would be better.
```suggestion
    public static boolean isAltPressed() {
```